client/src/featureform/serving.py

```diff
@@ -99,7 +99,7 @@ def _local_training_set(self, trainingSetName, trainingSetVariant):
                 df.set_index(feature_row[9])
```

Comment:
Just out of curiosity, what do the numbers represent? the `9`, `10`, `11` inside the `feature_row`.

Reply:
9 is entity name, believe 11 is the value column name and 11 is the timestamp column name. Part of ksshirajas task this week is to refactor this so were not using magic numbers anymore